app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def manage_assets_btn_text
```

Comment:
I don't think you need this manage_assets_btn_text helper method.  Since there are separate views for monograph and section, you can just insert the translation "t" method directly into the views.